apps/web/lib/hooks/useIsThemeSupported.ts

```diff
@@ -0,0 +1,14 @@
+"use client";
+
+import { usePathname } from "next/navigation";
+
+const THEME_UNSUPPORTED_ROUTES = ["/auth/setup"];
```

Comment:
<img width="495" alt="Screenshot 2025-01-18 at 6 03 24 PM" src="https://github.com/user-attachments/assets/b68e0e71-a739-4b20-bda4-5f0941158560" />

`auth/setup` is the only route with false value for `isThemeSupported`